### ISSUE TITLE
Getting timeout error for newrelic monitors

### ISSUE THREAD

**Issue by aujjwal-smarsh** [newrelic/terraform-provider-newrelic#1495]
I am running terraform version = 0.12.29.
This is the pipeline code I am running:
```yml
          terraform init
          terraform validate
          terraform apply -auto-approve -parallelism=4
```

Initially I didn't have added parallelism but after going through the ISSUE https://github.com/newrelic/terraform-provider-newrelic/issues/884 I added the flag and checked with the values 2,3,4,7,8,9,10,11,12,20,50 but none work.

Either I receive:
```sh
https://synthetics.newrelic.com/synthetics/api/v4/monitors/xxx-xxx-xxx-xxx-xxx giving up after 4 attempt(s)
```

OR

```sh
Error: Validation Error on modules/my_alerts/nrql_monitor/main.tf line 1, in resource "newrelic_nrql_alert_condition" "simple_nrql_monitor_condition":
   1:   resource "newrelic_nrql_alert_condition" "simple_nrql_monitor_condition" {
```
The above error is 10 times and only for alert.

**Comment by rdhar** [comment]
This is likely to be addressed as part of developments in #1453.

**Comment by kidk** [comment]
We will close this for now. If this is still an issue for you please open a new ticket.
